packages/daemon/src/app/controller/plugin.ts

```diff
@@ -77,23 +89,16 @@ export class PluginController extends BaseEventController {
    */
   @get('/:id')
   public async get() {
-    const plugin = await this.pluginManager.findById(this.ctx.params.id);
-    if (plugin) {
-      this.ctx.success(plugin);
-    } else {
-      this.ctx.throw(HttpStatus.NOT_FOUND, 'no plugin found');
-    }
+    const plugin = await this.fetchPluginByIdPrefix(this.ctx.params.id);
```

Comment:
I'm -1 on this change, we would keep the id to be the string that matches the actual id in the database, why not just implementing this in CLI?